test/test_check_branch.py

```diff
@@ -70,3 +70,18 @@ def expect_exception(branch, deployment, message):
                              'prod',
                              "Branch 'feature/foo' cannot be deployed to 'prod', "
                              "only one of {'sandbox'} or personal deployments.")
+
+        supported_vars = ['CI_COMMIT_REF_NAME', 'GITHUB_BASE_REF', 'GITHUB_HEAD_REF']
+
+        for active_env_var in supported_vars:
```

Comment:
The  variable names are inconsistent (env_var vs just _var or _vars) and overly long.